### PR TITLE
Assorted docs cleanups

### PR DESCRIPTION
**Pull request by dfarrell07**

### CODE REVIEW

**Comment by mangelajo** [comment]
oops, it's conflicting now. Why draft?

File: src/content/architecture/route-agent/_index.en.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ responsible for setting up VxLAN tunnels and routing the cross cluster traffic
 from the node to the cluster’s active Gateway Engine which subsequently sends the
 traffic to the destination cluster.
 
-When running on the same node as the active Gateway Engine, the Route Agent creates
-a VxLAN VTEP interface to which the Route Agent instances running on the other worker
-nodes in the local cluster connect by establishing a VXLAN tunnel with the VTEP of
-the active Gateway Engine node. The MTU of VxLAN tunnel is configured based on the MTU
+When running on the same node as the active Gateway Engine, Route Agent creates
+a VxLAN VTEP interface to which Route Agent instances running on the other worker
```
Comment (nyechiel) [inline src/content/architecture/route-agent/_index.en.md:12]: Why VxLAN? https://tools.ietf.org/html/rfc7348

File: src/content/architecture/globalnet/_index.en.md [hunk 2]
```diff
@@ -3,32 +3,31 @@ title = "Globalnet Controller"
...
 This is largely problematic because most actual deployments use the default CIDRs for a cluster so every cluster ends up using the same
-CIDRs. Changing CIDRs on existing clusters is a very disruptive process and requires a cluster restart. So submariner needs a way to allow
+CIDRs. Changing CIDRs on existing clusters is a very disruptive process and requires a cluster restart. So Submariner needs a way to allow
 clusters with overlapping CIDRs to connect together.
 
 ## Architecture
 
-To support overlapping CIDRs in clusters connected through submariner, submariner has a component called Global Private Network, GlobalNet
-(`globalnet`). This GlobalNet is a virtual network specifically to support submariner's multi-cluster solution with a global CIDR. Each
+To support overlapping CIDRs in clusters connected through Submariner, Submariner has a component called Global Private Network, GlobalNet
```
Comment (nyechiel) [inline src/content/architecture/globalnet/_index.en.md:25]: We currently have both "Globalnet" and "GlobalNet" being used across different pages. Should we agree on one?

File: src/content/architecture/route-agent/_index.en.md (hunk 1 above)
Comment (dfarrell07) [reply src/content/architecture/route-agent/_index.en.md]: Yep, good point, I changed it to VXLAN everywhere in this file.

**Comment by dfarrell07** [comment]
> oops, it's conflicting now. Why draft?

I was planning on collecting more doc cleanup commits under this PR, but yeah maybe would be better to go ahead and get this group in.

File: src/content/architecture/globalnet/_index.en.md [hunk 3]
```diff
@@ -3,32 +3,31 @@ title = "Globalnet Controller"
...
+
 ## Introduction
 
-Submariner is a tool built to connect overlay networks of different
-Kubernetes clusters. These clusters can be on different public clouds
-or on-premise. An important use case for Submariner is to connect disparate
-independent clusters into a single cohesive multi-cluster fleet.
+Submariner is a tool built to connect overlay networks of different Kubernetes clusters. These clusters can be on different public clouds or
+on-premise. An important use case for Submariner is to connect disparate independent clusters into a single cohesive multi-cluster cluster
+set.
```
Comment (tpantelis) [inline src/content/architecture/globalnet/_index.en.md:11]:
```suggestion
on-premise. An important use case for Submariner is to connect disparate independent clusters into a single cohesive set.
```

File: src/content/architecture/globalnet/_index.en.md [hunk 4]
```diff
@@ -3,32 +3,31 @@ title = "Globalnet Controller"
...
 
-However, by default, a limitation of submariner is that it doesn't handle
-overlapping CIDRs (ServiceCIDR and ClusterCIDR) across these clusters.
-Each cluster must use distinct CIDRs that don't conflict or overlap with any
-other cluster that is going to be part of cluster fleet.
+However, by default, a limitation of Submariner is that it doesn't handle overlapping CIDRs (ServiceCIDR and ClusterCIDR) across these
+clusters. Each cluster must use distinct CIDRs that don't conflict or overlap with any other cluster that is going to be part of the
```
Comment (tpantelis) [inline src/content/architecture/globalnet/_index.en.md:14]:
```suggestion
However, by default, a limitation of Submariner is that it doesn't handle overlapping CIDRs (ServiceCIDR and ClusterCIDR) across clusters.
Each cluster must use distinct CIDRs that don't conflict or overlap with any other cluster that is going to be part of the
```

File: src/content/architecture/globalnet/_index.en.md (hunk 2 above)
Comment (tpantelis) [inline src/content/architecture/globalnet/_index.en.md:25]:
```suggestion
To support overlapping CIDRs in connected clusters, Submariner has a component called Global Private Network, GlobalNet
```

Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md:25]: I added a commit to propose standardizing on "Globalnet" via the Docs Style Guide's Word List. Good feedback, thanks!

File: src/content/architecture/globalnet/_index.en.md (hunk 3 above)
Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md:11]: Maybe "single cluster set"? I was trying to use SIG MC's new term.

File: src/content/architecture/globalnet/_index.en.md [hunk 5]
```diff
@@ -3,32 +3,31 @@ title = "Globalnet Controller"
...
+cluster set.
```
Comment (nyechiel) [inline src/content/architecture/globalnet/_index.en.md:15]:
```suggestion
clusterset.
```

File: src/content/architecture/globalnet/_index.en.md (hunk 2 above)
Comment (nyechiel) [inline src/content/architecture/globalnet/_index.en.md:25]:
```suggestion
To support overlapping CIDRs in clusters connected through Submariner, Submariner has a component called Global Private Network, Globalnet
```

File: src/content/architecture/globalnet/_index.en.md [hunk 6]
```diff
@@ -3,32 +3,31 @@ title = "Globalnet Controller"
...
-CIDRs. Changing CIDRs on existing clusters is a very disruptive process and requires a cluster restart. So submariner needs a way to allow
+CIDRs. Changing CIDRs on existing clusters is a very disruptive process and requires a cluster restart. So Submariner needs a way to allow
 clusters with overlapping CIDRs to connect together.
 
 ## Architecture
 
-To support overlapping CIDRs in clusters connected through submariner, submariner has a component called Global Private Network, GlobalNet
-(`globalnet`). This GlobalNet is a virtual network specifically to support submariner's multi-cluster solution with a global CIDR. Each
+To support overlapping CIDRs in clusters connected through Submariner, Submariner has a component called Global Private Network, GlobalNet
+(`globalnet`). This GlobalNet is a virtual network specifically to support Submariner's multi-cluster solution with a global CIDR. Each
```
Comment (nyechiel) [inline src/content/architecture/globalnet/_index.en.md:26]:
```suggestion
(`globalnet`). This Globalnet is a virtual network specifically to support Submariner's multi-cluster solution with a global CIDR. Each
```

**Review by nyechiel — COMMENTED** [review]
I wonder if we should also use and define "clusterset" (as per KEP 1645)

File: src/content/architecture/globalnet/_index.en.md (hunk 2 above)
Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md]: Done in the latest, thanks!

Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md]: Also updated to "Globalnet" in this file.

File: src/content/architecture/globalnet/_index.en.md (hunk 3 above)
Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md]: Made an attempt in a new commit.

File: src/content/architecture/globalnet/_index.en.md (hunk 4 above)
Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md]: Done, thanks!

File: src/content/architecture/globalnet/_index.en.md (hunk 2 above)
Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md]: Did all of these in the latest.

File: src/content/architecture/globalnet/_index.en.md (hunk 6 above)
Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md]: Did all of these in the latest.

File: src/content/architecture/globalnet/_index.en.md (hunk 5 above)
Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md]: "ClusterSet" maybe?

Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md]: Done, thanks!

**Review by nyechiel — COMMENTED** [review]
"clusterset" is being used extensively in the KEP doc.. but "ClusterSet" was the name folks voted about. Not sure what is the right one to use.

**Comment by dfarrell07** [comment]
> "clusterset" is being used extensively in the KEP doc.. but "ClusterSet" was the name folks voted about. Not sure what is the right one to use.

I think they will end up having to follow the K8s style of CamelCase for the K8s object. Also I think the GDoc is informal and the poll is the key thing.

**Comment by dfarrell07** [comment]
I clarified my understanding of ClusterSet usage in the Word List.

File: src/content/architecture/globalnet/_index.en.md [hunk 7]
```diff
@@ -3,33 +3,29 @@ title = "Globalnet Controller"
...
 +++
+
 ## Introduction
 
-Submariner is a tool built to connect overlay networks of different
-Kubernetes clusters. These clusters can be on different public clouds
-or on-premise. An important use case for Submariner is to connect disparate
-independent clusters into a single cohesive multi-cluster fleet.
+Submariner is a tool built to connect overlay networks of different Kubernetes clusters. These clusters can be on different public clouds or
+on-premise. An important use case for Submariner is to connect disparate independent clusters into a ClusterSet.
```
Comment (nyechiel) [inline src/content/architecture/globalnet/_index.en.md:10]:
```suggestion
on-premises. An important use case for Submariner is to connect disparate independent clusters into a ClusterSet.
```

File: src/content/contributing/website/style_guide.md [hunk 8]
```diff
@@ -12,25 +12,28 @@ contribution and review process.
...
 A list of Submariner-specific terms and words to be used consistently across
 the site.
 
 Term | Usage
 :--- | :----
-Submariner | The project name *Submariner* should always be capitalized.
 Admiral | The project name *Admiral* should always be capitalized.
-Lighthouse | The project name *Lighthouse* should always be capitalized.
+ClusterSet | The K8s object *ClusterSet* proposed in KEP1645 should always be CamelCase.
+Cluster set | The words "cluster set" should be used as a term for a group of clusters, but not the proposed K8s object.
```
Comment (nyechiel) [inline src/content/contributing/website/style_guide.md:24]:
```suggestion
Cluster set | The words "cluster set" should be used as a term for a group of clusters, but not the proposed Kubernetes object.
```

File: src/content/contributing/website/style_guide.md [hunk 9]
```diff
@@ -12,25 +12,28 @@ contribution and review process.
...
 
 A list of Submariner-specific terms and words to be used consistently across
 the site.
 
 Term | Usage
 :--- | :----
-Submariner | The project name *Submariner* should always be capitalized.
 Admiral | The project name *Admiral* should always be capitalized.
-Lighthouse | The project name *Lighthouse* should always be capitalized.
+ClusterSet | The K8s object *ClusterSet* proposed in KEP1645 should always be CamelCase.
```
Comment (nyechiel) [inline src/content/contributing/website/style_guide.md:23]:
```suggestion
ClusterSet | The Kubernetes object *ClusterSet* proposed in KEP1645 should always be CamelCase.
```

**Review by tpantelis — APPROVED** [review]

File: src/content/contributing/website/style_guide.md (hunk 8 above)
Comment (dfarrell07) [reply src/content/contributing/website/style_guide.md]: Amended that commit to fix, thanks.

File: src/content/contributing/website/style_guide.md (hunk 9 above)
Comment (dfarrell07) [reply src/content/contributing/website/style_guide.md]: Amended that commit to fix, thanks.

File: src/content/architecture/globalnet/_index.en.md (hunk 7 above)
Comment (dfarrell07) [reply src/content/architecture/globalnet/_index.en.md]: Amended the relevant cleanup commit to add this fix, thanks.

**Review by nyechiel — APPROVED** [review]
LGTM

**Pull request merged**